internal/handlers/sqlite/msg_create.go

```diff
@@ -79,6 +70,48 @@ func (h *Handler) MsgCreate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		return nil, err
 	}
 
+	params := backends.CreateCollectionParams{
+		Name: collectionName,
+	}
+
+	var capped bool
+	if v, _ := document.Get("capped"); v != nil {
+		capped, err = commonparams.GetBoolOptionalParam("capped", v)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	if h.EnableOplog && capped {
+		var size any
+
+		size, err = document.Get("size")
+		if err != nil {
+			return nil, err
```

Comment:
It shouldn't be internal error right? Also we should avoid checking error https://github.com/FerretDB/FerretDB/blob/f1b8354962ce641343836d819b8c29fab19eeb87/internal/types/document.go#L286-L287

Reply:
Nice catch! It should be `the 'size' field is required` error, yes! I'll add a test and a fix.